src/App.ts

```diff
@@ -1,10 +1,11 @@
-/* eslint-disable @typescript-eslint/explicit-member-accessibility, @typescript-eslint/strict-boolean-expressions */
 import { Agent } from 'http';
 import { SecureContextOptions } from 'tls';
 import util from 'util';
 import { WebClient, ChatPostMessageArguments, addAppMetadata, WebClientOptions } from '@slack/web-api';
 import { Logger, LogLevel, ConsoleLogger } from '@slack/logger';
 import axios, { AxiosInstance } from 'axios';
+import allSettled from 'promise.allsettled';
```

Comment:
I also want to add that I saw issues with removing require from `allSettled` last time too, but I don't remember what that issue was off the top of my head